configuration/frontend/base-config.json

```diff
@@ -31,5 +31,17 @@
       "generalPatientNoteUuid": "3cd9d956-26fe-102b-80cb-0017a47871b2",
       "midUpperArmCircumferenceUuid": "4326b04b-3158-417a-bb8d-ad022295b0f4"
     }
+  },
+  "@openmrs/esm-patient-medications-app": {
+    "durationUnitsConcept": "3cd706b8-26fe-102b-80cb-0017a47871b2",
```

Comment:
What does "durationUnitsConcept" represent?  What does it need configuration client-side in the first place, and are you sure this is the right value?  There is already a global property set (which we do in config-pihemr) for order.durationUnitsConceptUuid.  Is this something different?  The value we are setting for the GP is 19cd6ff8-052b-46e7-96ef-91fccede9565, which is different from what you have configured here...

Reply:
You are right, this is definitely wrong, because it looks like the same one used for days before... I had to do some cut-and-pasting of this whole config at one point, I probably took an old version I was using.